src/yearn.ts

```diff
@@ -24,6 +24,8 @@ import { ZapperService } from "./services/zapper";
 import { SdkError } from "./types/common";
 import { AssetServiceState } from "./types/custom/assets";
 
+const orig = JsonRpcSigner.prototype.sendTransaction;
```

Comment:
More descriptive name for this since it's not near where it's used anymore? maybe `originalSendTransactionFunction`? not sure